src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sources/streamfactories/FacebookPageStreamFactory.scala

```diff
@@ -17,7 +17,11 @@ class FacebookPageStreamFactory extends StreamFactory[FacebookPost] {
   override def createStream(streamingContext: StreamingContext): PartialFunction[ConnectorConfig, DStream[FacebookPost]] = {
     case ConnectorConfig("FacebookPage", params) =>
       val facebookAuth = FacebookAuth(params("appId"), params("appSecret"), params("accessToken"))
+      val pageIds = Option(params("pageIds")) match {
```

Comment:
We should raise an exception at the highest severity level to app insights so we're notified. I'll add that.